infrastructure/index.ts

```diff
@@ -434,6 +435,34 @@ const distributionArgs: aws.cloudfront.DistributionArgs = {
             defaultTtl: 0,
             minTtl: 0,
             maxTtl: 0,
+        },
+
+        // AI app, live, with caching handled by the app
+        {
+            ...baseCacheBehavior,
+            // allow all methods
+            allowedMethods: ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"],
+            cachedMethods: [
+                "GET", "HEAD", "OPTIONS",
+            ],
+            targetOriginId: aiAppDomain,
+            pathPattern: '/ai',
+            defaultTtl: 0,
+            minTtl: 0,
+            maxTtl: 0,
+        },
+        {
+            ...baseCacheBehavior,
+            // allow all methods
+            allowedMethods: ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"],
+            cachedMethods: [
+                "GET", "HEAD", "OPTIONS",
+            ],
+            targetOriginId: aiAppDomain,
```

Comment:
Running against my dev stack now, but do we need to add an `origin` for this also? I see there's one for `previewAIAppDomain`.

Reply:
Yeah, looks like it:

```
error updating CloudFront Distribution (E3GY8OM5MAPY0D): NoSuchOrigin: One or more of your cache behavior target origins or origin groups do not exist.
```